### PR TITLE
[stable/rocketchat] update to 1.3.1, upgrade mongo to latest helm chart

### PR DESCRIPTION
**Pull request by verwilst**
Signed-off-by: Bart Verwilst <bart@verwilst.be>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/rocketchat/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rocketchat
-version: 1.1.6
+version: 1.1.7
 appVersion: 1.2.3
```
Comment (srueg) [inline stable/rocketchat/Chart.yaml]: Please update the `appVersion` accordingly

**Review by srueg — CHANGES_REQUESTED** [review]

**Comment by geekgonecrazy** [comment]
What actual versions of mongo is this moving between?

**Comment by verwilst** [comment]
Both are 4.0.9, so no changes there.

**Pull request merged**